specifications/api/swagger.yaml

```diff
@@ -1788,17 +1788,29 @@ x-transactionMetadata: &transactionMetadata
   # propertyNames:
   #   pattern: '^[0-9]+$'
 
+x-encryptionMethod: &encryptionMethod
+  type: string
+  enum:
+    - basic
+
 x-encryptMetadata: &encryptMetadata
   description: |
-    If used then metadata in transaction is going to be encrypted
-    via AEAD scheme using ChaCha20 and Poly1305 (see RFC 7539).
-    PBKDF2 password stretching is used to get a 32-byte symmetric key.
-    PBKDF2 encryption using HMAC with the hash algorithm SHA512 is employed here.
-    The encrypted metadata is going to be stored in blockchain as a consequence.
+    If used then metadata in a transaction is going to be encrypted by
+    AES 256 using CBC mode which is a default method (called base).
+    PBKDF2 password stretching is used to get a 32-byte secret key and a 16-byte
+    initialization vector required in the cipher.
+    PBKDF2 encryption algorithm using HMAC with the SHA256 hash algorithm is employed,
+    and 10000 iterations to get key and iv pair are used.
+    Cipher algorithm uses 8-byte salt, PKCS#7 padding as specified in
+    https://datatracker.ietf.org/doc/html/rfc5652#section-6.3 is applied.
+    Only metadata value under `msg` field is encrypted. If `msg` field is missing error
+    will be emitted.
```

Comment:
As a user who is new to this feature, I'm finding this `description` section a little hard to follow!

For example, it refers to a field called `msg`. However, this object only has two fields: `enc` and `passphrase`.

From reading through the tests (`Api.TypesSpec`), I can see that `msg` is supposed to be the key in a `TxMetaMap`.

Is there any way that we can give a more complete description of how to use this feature? (Ideally, so that the user doesn't have to go hunting through CIP 83.)